### DISCUSSION TITLE
Listening to changes in the database

### DISCUSSION THREAD

**Discussion by Quwaysim** [tekartik/sqflite#794] [Ideas]
I was just wondering, since sqlite itself supports listening to changes in the db, is there any way to implement this in android? Is it feasible? If yes, is this in the sqflite package roadmap?

Thank you.

**Comment by alextekartik** [comment]
I'm not aware of any support for listening for changes in sqlite Android. Typically this should be done at higher level (i.e. by tracking for example changes in the tables to know which query should be run again). Such a change is not planned for sqflite.
